app/components/buttons/ToggleSwitch.js

```diff
@@ -0,0 +1,27 @@
+import "style/ToggleSwitch.less";
+import { Tooltip } from "shared";
+
+// enabled shows the switch as on or off.
+// disabled unable to trigger the onClick method.
+// enabledText and notEnabledText the tooltop contents when the switch is on/off.
+const ToggleSwitch = ({ enabled, onClick, disabled, enabledText, notEnabledText }) => (
+  <Tooltip
+    text={ enabled ? enabledText : notEnabledText }>
+    <div className="toggle-switch">
+      <div
+        className={
+          enabled ? "toggle-switch-enabled" : "toggle-switch-disabled"
```

Comment:
use `classNames` function which can be imported from `pi-ui` to use conditioned classes in a cleaner way